src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -191,74 +214,83 @@ class DropDownGroup extends React.Component {
                     state !== "exited" && !shouldOpenDownward;
 
                   return (
-                    <StyledGroupWrapper
-                      {...props}
-                      className={classNames(props.className, {
-                        "dropdown--open-upward": hasOpenUpwardClass,
-                        "dropdown--disabled": disabled
-                      })}
-                      tabIndex={disabled ? -1 : 0}
-                      aria-haspopup="listbox"
-                      aria-labelledby={hiddenLabelId}
-                      onKeyDown={this.onKeyDown}
-                      ref={this.groupWrapper}
-                    >
-                      <StyledGroup
-                        {...onClickListener}
-                        className={classNames(`dropdown--${size}`, {
-                          "dropdown--active": isOpen,
-                          "dropdown--border": variant === 0,
-                          "dropdown--no-border": variant === 1,
-                          "dropdown__label--disabled": disabled
+                    <Fragment>
+                      {this.getOuterLabel()}
```

Comment:
Hi @OleksandrKrupko, I have a few questions:

1. Is there an advantage to making this label driven by prop inheritance? It seems like a composition based solution would be more flexible 🤔

2. Are there multiple products with the requirement to display an outer label with the same design above the `DropDownGroup`? I could see us creating a `DropDownLabel` component for use in composition.

Reply:
Hmm, that's a good idea, just didn't want to separate this piece of code into its own component because it looked quite small for doing that. I think that the creation of some generic label is a great idea but it'll be better to do this refactoring in some other PR

So, sure,  I'll put this label into its own component

Reply:
Sounds good. Thank you!